CMakeLists.txt

```diff
@@ -53,17 +53,16 @@ STRING(REGEX REPLACE "#define[ ]+OPENSHOT_VERSION_MAJOR[ ]+([0-9]+);(.*)" "\\1"
 STRING(REGEX REPLACE "#define[ ]+OPENSHOT_VERSION_MINOR[ ]+([0-9]+);(.*)" "\\1" MINOR_VERSION "${LINE_MINOR}")
 STRING(REGEX REPLACE "#define[ ]+OPENSHOT_VERSION_BUILD[ ]+([0-9]+);(.*)" "\\1" BUILD_VERSION "${LINE_BUILD}")
 STRING(REGEX REPLACE "#define[ ]+OPENSHOT_VERSION_SO[ ]+([0-9]+);(.*)" "\\1" SO_VERSION "${LINE_SO}")
-set(PROJECT_VERSION "${MAJOR_VERSION}.${MINOR_VERSION}.${BUILD_VERSION}")
+
+################### SETUP PROJECT ###################
+PROJECT(libopenshot VERSION ${MAJOR_VERSION}.${MINOR_VERSION}.${BUILD_VERSION})
```

Comment:
I went looking to see if I could reduce that `cmake_minimum_required()` call earlier in this source to something compatible with the ancient version used on the build server, but the `VERSION` argument to `project()` — the lack of which is a source of major complaining whenever those of us using a modern cmake try to build libopenshot — was introduced in 3.0.  I really don't want to give that up jjust to remain compatible with an incredibly out of date version.

Reply:
Mr. Thomas has been MIA for some time now. I'll let him know about the issue with the build servers once he resurfaces. :smile: